opentreemap/treemap/css/sass/partials/pages/_modeling.scss

```diff
@@ -397,6 +397,17 @@
             }
         }
     }
+
+    .leaflet-loading-control {
+        background: #fff;
+        padding: 5px 10px;
+        font-size: 12pt;
```

Comment:
I kind of like the flat look but I guess it makes sense to match the other leaflet controls instead. Updated.